chrome/content/zotero/zoteroPane.js

```diff
@@ -1901,6 +1923,35 @@ var ZoteroPane = new function()
 		}
 	});
 	
+	this.editSelectedFeed = Zotero.Promise.coroutine(function* () {
+		if (!this.collectionsView.selection.count) return;
+		
+		let feed = this.collectionsView.getRow(this.collectionsView.selection.currentIndex).ref;
+		let data = {
+			url: feed.url,
+			title: feed.name,
+			ttl: feed.refreshInterval,
+			cleanAfter: feed.cleanupAfter
+		};
+		
+		window.openDialog('chrome://zotero/content/feedSettings.xul', 
+			null, 'centerscreen, modal', data);
+		if (data.cancelled) return;
+		
+		feed.name = data.title;
+		feed.refreshInterval = data.ttl;
+		feed.cleanupAfter = data.cleanAfter;
+		yield feed.save({skipEditCheck: true});
```

Comment:
`.saveTx` and I don't think we need the `skipEditCheck` any more (this dates back to when feeds extended collections, rather than libraries)